### DISCUSSION TITLE
Keep popover opened when opening a nested Dialog

### DISCUSSION THREAD

**Discussion by vctormb** [floating-ui/floating-ui#2123] [Q&A] [answered]
Hello 👋 
First I'd like to thank all of you for this amazing library!

I have a case where I need to show up a Popover when hovering the trigger element. Inside the Popover I have a button where it opens up a Dialog. When the Dialog opens up I need to keep the Popover opened and when closing the Dialog, the Popover should still keep opened, but if I interact with the Popover again it should close the Popover when moving out the safe area.

Does anyone know if is it possible to achieve this result using floating-ui?

Example: https://codesandbox.io/s/festive-payne-0npgd6?file=/src/App.tsx

Thanks!

**Comment by atomiks** [comment, marked as answer]
Disabling the `useHover` hook (`enabled` prop) when the Dialog opens will prevent it from closing, but I'm not sure if it achieves the second part of what you want when the Dialog closes.

**Reply by vctormb** [reply]
It worked! Thanks @atomiks for the suggestion! I've updated the code sandbox
